manifests/cacert.pp

```diff
@@ -22,17 +22,20 @@
     $ca_location,
     $ca_file,
 ) {
-    file { $ca_location:
-        ensure => directory,
-        owner  => 'root',
-        group  => 'root',
-    }
-    file { "${ca_location}/${ca_file}":
-        owner   => 'root',
-        group   => 'root',
-        mode    => '0600',
-        links   => 'follow',
-        source  => $ca_source,
-        require => File[$ca_location],
+    # not perfect, but if this file is already defined don't do it again
+    if ! defined (File["${ca_location}/${ca_file}"]) {
```

Comment:
As noted in the issue we have a module called "base" that causes this file
to land.  This "base" module isn't used for the current prod lightblue
deployment, it is used when we deploy in our datacenter only.

On Wed, Aug 19, 2015 at 9:18 AM, Dennis Crissman <notifications@github.com>
wrote:

> In manifests/cacert.pp
> <https://github.com/lightblue-platform/lightblue-puppet/pull/151#discussion_r37412621>
> :
>
> > @@ -22,17 +22,20 @@
> >      $ca_location,
> >      $ca_file,
> >  ) {
> > -    file { $ca_location:
> > -        ensure => directory,
> > -        owner  => 'root',
> > -        group  => 'root',
> > -    }
> > -    file { "${ca_location}/${ca_file}":
> > -        owner   => 'root',
> > -        group   => 'root',
> > -        mode    => '0600',
> > -        links   => 'follow',
> > -        source  => $ca_source,
> > -        require => File[$ca_location],
> > +    # not perfect, but if this file is already defined don't do it again
> > +    if ! defined (File["${ca_location}/${ca_file}"]) {
>
> What would cause this to already be defined in the puppet file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-puppet/pull/151/files#r37412621>
> .
>